charts/cadc-tap/templates/tap-deployment.yaml

```diff
@@ -82,23 +82,20 @@ spec:
                 -Dtap.password=$TAP_DB_PASSWORD
                 -Dtap.url=jdbc:postgresql://{{ .Values.config.pg.host }}/{{ .Values.config.pg.database }}
                 -Dtap.maxActive=100
-                -Dca.nrc.cadc.auth.Authenticator=ca.nrc.cadc.sample.AuthenticatorImpl
                 {{- end }}
                 {{- if eq .Values.config.backend "qserv" }}
                 -Dqservuser.username=qsmaster
                 -Dqservuser.password=
                 -Dqservuser.driverClassName=com.mysql.cj.jdbc.Driver
                 -Dqservuser.url=jdbc:mysql://{{ .Values.config.qserv.host }}/{{ .Values.config.qserv.jdbcParams }}
                 -Dqservuser.maxActive=100
-                -Dca.nrc.cadc.auth.Authenticator=org.opencadc.tap.impl.AuthenticatorImpl
                 {{- end }}
-                -Dca.nrc.cadc.reg.client.RegistryClient.local=true
                 -Dgafaelfawr_url={{ .Values.global.baseUrl }}/auth/api/v1/user-info
                 -Dgcs_bucket={{ .Values.config.gcsBucket }}
                 -Dgcs_bucket_url={{ .Values.config.gcsBucketUrl }}
                 -Dgcs_bucket_type={{ .Values.config.gcsBucketType }}
                 -Dbase_url={{ .Values.global.baseUrl }}
-                -Dca.nrc.cadc.util.PropertiesReader.dir=/etc/creds/
+                -Dca.nrc.cadc.util.PropertiesReader.dir=/config/
```

Comment:
I dislike creating new top-level directories.  But that's probably just me being a Unix dinosaur.

Reply:
I did try a different directory which doesn't seem to work. I think because the cadc TAP service expects to find the config there. It may be possible to convince it to use a different directory, but not sure how complicated it will be.
Any objections to going with this for now, and perhaps revisit later as a separate task?